.circleci/config.yml

```diff
@@ -221,11 +222,13 @@ jobs:
           drush vset -y opensearch_search_autocomplete_method 'facets'
           drush vset -y ting_openlist_prefix test
           drush vset -y ting_openlist_wsdl 'https://test.openlist.ddbcms.dk/?wsdl'
+          drush vset -y ting_library_code 100200
           drush vset -y user_consent_activate true
           drush vset -y loan_history_store_title ''
           drush vset -y loan_history_store_title_first_time ''
           drush php-eval 'variable_set("loan_history_store_description_first_time", array("value" => ""));'
           drush php-eval 'variable_set("loan_history_store_description", array("value" => ""));'
+          sudo chown -R 33.33 /var/www/html/sites/default/files
```

Comment:
This should be '33:33' instead of '33.33'. Backward compatibility supports '.' only as long as no ambiguity results.
Also, are we sure 33 is always the uid we want? (it's a fairly obscure notation, IMO)